Dockerfile

```diff
@@ -12,7 +12,6 @@ RUN dnf install -y python3 python3-requests && \
     curl -Lfv https://raw.githubusercontent.com/openstack/tripleo-repos/master/plugins/module_utils/tripleo_repos/main.py | python3 - -b master current-tripleo && \
     sed -i 's/>=.*$//g' /tmp/${PKGS_LIST} && \
     prepare-image.sh && \
-    mkdir -p /etc/ironic-python-agent && \
```

Comment:
Used on line 22

Reply:
yep
this is not removed, it's just moved inside prepare-image.sh, called in the precedent line